Core/ExposureNPC.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Terraria;
+using Terraria.ModLoader;
+
+namespace StarlightRiver.Core
+{
+    public class ExposureNPC : GlobalNPC
+    {
+        public int ExposureAdd;
```

Comment:
Having these be hardcoded class specific is fine for now, but in the future for compatability it may be wise to re-design this as a dictionary-based system with DamageClass keys to be able to re-use the system for arbitrary damage types.